sos/plugins/peripety.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2018 Red Hat, Inc., Jake Hunsaker <jhunsake@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+from re import match
+import glob
+
+
+class Peripety(Plugin, RedHatPlugin):
+    '''Peripety Storage Event Monitor'''
+
+    packages = ('peripety',)
+    services = ('peripetyd',)
+
+    def setup(self):
+        self.add_copy_spec('/etc/peripety.conf')
+
+        forbid_reg = [
+            'vd.*',
+            'sr.*'
+        ]
```

Comment:
Do we need to split this over multiple lines? Looks to be pretty short strings, or do we expect further additions in future?

Reply:
Yep - I imagine this list will be built out in the future.